movie-showcase/src/routes/+layout.svelte

```diff
@@ -0,0 +1,23 @@
+<a href="/"><div class="homebutton" /></a>
```

Comment:
This is really, really basic for a home button. Even though the styling was not the focus, this could use a liiiiitle bit more attention, I think. Even a simple text saying this is the home button would be a massive improvement, because now it is just a 10px red block 😄.